specification/network/resource-manager/Microsoft.Network/stable/2017-11-01/networkWatcher.json

```diff
@@ -61,7 +61,7 @@
           {
             "name": "parameters",
             "in": "body",
-            "required": true,
+            "required": false,
```

Comment:
This seems like a significant change; why are the parameters no longer required?